gtsfm/runner/frontend_runner.py

```diff
@@ -0,0 +1,70 @@
+"""Runner for the front-end only (when back-end results are not desired or required).
+
+Author: John Lambert
+"""
+from typing import Dict, Tuple
+
+import dask
+from dask.delayed import Delayed
+
+from gtsfm.feature_extractor import FeatureExtractor
+from gtsfm.loader.loader_base import LoaderBase
+from gtsfm.two_view_estimator import TwoViewEstimator
+
+
+def run_frontend(
+    loader: LoaderBase, feature_extractor: FeatureExtractor, two_view_estimator: TwoViewEstimator
+) -> Tuple[Delayed, Delayed]:
+    """Creates the front-end computation graph, and then runs it.
+
+    Note: Copied from SceneOptimizer class, without back-end code.
+
+    Args:
+        loader: image loader.
+        feature_extractor: feature extractor module to use.
+        two_view_estimator: two-view estimator module to use.
+
+    Returns:
+        keypoints_list_results: detected keypoints for each image.
```

Comment:
why the _results suffix ? 
Actually we don't even need variable names for returns.

Reply:
Sure, I've removed the `_results` suffix now. Good suggestion.